actions.go

```diff
@@ -67,6 +75,35 @@ func (c *Action) IssueCommand(cmd *Command) {
 	fmt.Fprintln(c.w, cmd.String())
 }
 
+// IssueFileCommand issues a new GitHub actions Command using environment files.
+//
+// https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#environment-files
+//
+// The TypeScript equivalent function:
+//
+// https://github.com/actions/toolkit/blob/4f7fb6513a355689f69f0849edeb369a4dc81729/packages/core/src/file-command.ts#L10-L23
+//
+// IssueFileCommand currently ignores the 'CommandProperties' field provided
+// with the 'Command' argument as it's scope is unclear in the current
+// TypeScript implementation.
+func (c *Action) IssueFileCommand(cmd *Command) error {
+	e := strings.ReplaceAll(cmd.Name, "-", "_")
+	e = strings.ToUpper(e)
+	e = "GITHUB_" + e
+
+	w, err := os.OpenFile(os.Getenv(e), os.O_APPEND|os.O_WRONLY, os.ModeAppend)
+	if err != nil {
+		return fmt.Errorf(errFileCmdFmt, err)
+	}
+
```

Comment:
Need to defer closing the file. Also need to sync and flush with error handling. Alternatively, use `ioutil.WriteFile`

Reply:
Oh missed it! I think `ioutil.WriteFile` sounds cleaner. I'll fix this!